src/views/RecordingPreviewer/RecordingPreviewer.tsx

```diff
@@ -113,25 +118,17 @@ export function RecordingPreviewer() {
         </>
       }
     >
-      <Allotment defaultSizes={[1, 1]}>
-        <Allotment.Pane>
-          <RequestsSection
-            groups={groups}
-            proxyData={proxyData}
-            noDataElement={<EmptyMessage message="The recording is empty" />}
-            selectedRequestId={selectedRequest?.id}
-            onSelectRequest={setSelectedRequest}
-          />
-        </Allotment.Pane>
-        {selectedRequest !== null && (
-          <Allotment.Pane minSize={300}>
-            <Details
-              selectedRequest={selectedRequest}
-              onSelectRequest={setSelectedRequest}
-            />
-          </Allotment.Pane>
-        )}
-      </Allotment>
+      {!isLoading && settings?.recorder.enableBrowserRecorder && (
```

Comment:
We check `isLoading` here because otherwise there will be a flicker when switching between recordings (probably due to the way we do `setProxyData([])` in the `useEffect`-disposer).